assets/js/data/cart/actions.js

```diff
@@ -85,17 +85,17 @@ export function receiveCartItem( response = {} ) {
 
 /**
  * Returns an action object to indicate if the specified cart item
- * is being updated; i.e. removing, or changing quantity.
+ * is being removing.
  *
- * @param {string} cartItemKey Cart item being updated.
- * @param {boolean} isQuantityPending Flag for update state; true if API request is pending.
+ * @param {boolean} isPendingDelete Flag for update state; true if API request is pending.
+ * @param {string} cartItemKey Cart item being deleted.
  * @return {Object} Object for action.
  */
-export function itemQuantityPending( cartItemKey, isQuantityPending ) {
+export function itemPendingDelete( isPendingDelete, cartItemKey ) {
```

Comment:
Switching the order of this function is a personal preference, in my opinion, flag function or flag parameters (`shouldCondition` or `shouldAction`) should have either:
1- the flag as the first parameter.
2- the param as an object and the flag as a named value in it, so that you know what is true or not, example:
```js
deleteStatus( { isDeleting: true } )
```
or
```js
isDeletingItem( true, itemKey )
```

Reply:
I didn't see that you had left this comment in my original review (because the comment is on outdated code).  Just want to note, that I don't think personal preference should play a role here (you can read my inline review comment for more details).

Reply:
That’s why I commented on it here so that we can discuss it, personal preferences should be discussed before making them a general rule since we all share this code.